config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -41,6 +41,56 @@ presubmits:
             memory: "6Gi"
         securityContext:
           privileged: true
+  - name: pull-kubernetes-node-e2e-containerd-decorate
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pull-kubernetes-node-e2e-containerd-decorate
+    always_run: false
+    optional: true
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    decoration_config:
+      timeout: 65m
+    spec:
+      serviceAccountName: node-e2e-tests
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230727-ea685f8747-master
+        command:
+        - runner.sh
+        args:
+        - kubetest2
+        - noop
+        - --test=node
```

Comment:
I didn't find anything about this argument in https://gist.github.com/dims/c1296f8ed42238baea0a5fcae45f4cf4. Is it saying to run e2e_node instead of e2e/node?

Reply:
I was mostly looking at the examples in sig-node around this migration.